### PR TITLE
Update recipe to nvcomp 3.0.2.

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

**Comment by bdice** [comment]
@adibbley Would you be able to review this (and merge if you approve)?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,19 +12,20 @@ package:
...
+    sha256: 0359b7fcc4e86f7547b2e80b564305daa3657f3093272d52d1c3382201850c4d  # [aarch64 and (cuda_compiler_version or "None").startswith("12")]
+    sha256: 42aa084c6092d816cb49ca8a80bb7f9f5843494d7582c20d8dd734bbfb0d2732  # [win and (cuda_compiler_version or "None").startswith("11")]
+    sha256: 8831a648dedbd884bbd81ecd604ac52902da27041af71620b4c95f62c388c772  # [win and (cuda_compiler_version or "None").startswith("12")]
   # Include cudart license because it is statically linked.
   - url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/LICENSE.txt
     fn: cudart_LICENSE.txt
     sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
 
 build:
   number: 3
```
Comment (adibbley) [inline recipe/meta.yaml]: I think we just need to reset the build number
```suggestion
  number: 0
```

**Comment by bdice** [comment]
@adibbley Good catch on the build number, thanks. This should be ready for review/merge again.

**Review by adibbley — APPROVED** [review]

**Pull request merged**